scripts/variantstore/wdl/extract/summarize_task_monitor_logs_from_file.py

```diff
@@ -0,0 +1,204 @@
+import argparse
+import sys
+import re
+import os
+from google.cloud import storage
+
+
+MaxCpu = -100.0
+MaxMem = -100.0
+MaxMemPct = -100.0
+MaxDisk = -100.0
+MaxDiskPct = -100.0
+
+
+def parse_monitoring_log_files(mlog_file, output_file):
+    with open(output_file, 'w') as output, open(mlog_file) as input:
+        header = f"Total Mem\tMax Mem Used\tMax Mem Used (%)\tTotal Disk\tMax Disk Used\tMax Disk Used (" \
+                 f"%)\tTask\tShard\tFile\n"
+        output.write(header)
+
+        for path in input:
+            client = storage.Client()
```

Comment:
I believe `client` can / should be pulled above the loop as I don't think you need to make a new one for each log file